portal-api/src/drivers/db-client.ts

```diff
@@ -10,3 +10,5 @@ export const getCollection = async <CollectionName extends MongoDbCollectionName
   await mongoDbClient.getCollection<CollectionName>(collectionName);
 
 export const close = async () => await mongoDbClient.close();
+
+export default mongoDbClient;
```

Comment:
Am somewhat tempted to go through every import of this file & change so they all use the default export

Reply:
Regret it but here we are https://github.com/UK-Export-Finance/dtfs2/pull/3213